src/pages/Test/Test.tsx

```diff
@@ -12,6 +14,16 @@ const Input = styled('input')({
   display: 'none',
 });
 
+const createSubscriptionData: CreateSubscriptionData = {
+  name: 'Adobe Photoshop',
+  description: 'For photo editing',
+  accessRights: ['Admin', 'Teacher'],
+  credentials: 'test123',
+  expiry: '2023-06-01',
```

Comment:
@jasonqiu212  will push a fix later for the expiry date. The date it accepts now will be slightly different. Ask @jasonqiu212 for the new date requirements in the backend and make the necessary changes